pkgversion/pkgversion.py

```diff
@@ -27,7 +28,11 @@ def get_git_repo_dir():
     repo_dir = repo_dir.strip()
     if not repo_dir:
         repo_dir = None
-    return repo_dir
+
+    country_code, encoding = locale.getdefaultlocale()
+    encoding = encoding if encoding else 'UTF-8'
+
+    return repo_dir.decode(encoding) if repo_dir and encoding else repo_dir
```

Comment:
It is better ensure, that `repo_dir` is not returned as byte string in Python 3, even if there's no encoding